components/mediation-admin/org.wso2.carbon.mediation.templates.common/src/main/java/org/wso2/carbon/mediation/templates/common/TemplateInfo.java

```diff
@@ -23,6 +23,8 @@
     private boolean enableStatistics;
     private boolean enableTracing;
     private String description;
+    private boolean isDeployedFromCApp = false;
+    private boolean isEdited = false;
```

Comment:
Boolean will be initialized to 'false' by default. No need of explicitly set it.